src/components/PaginationItem/PaginationItem.tsx

```diff
@@ -0,0 +1,24 @@
+interface Props {
+  page: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const PaginationItem: React.FC<Props> = ({
+  page,
+  currentPage,
+  setCurrentPage,
+}) => {
+  return (
+    <li className={`page-item ${currentPage === page ? 'active' : ''}`}>
+      <a
+        data-cy="pageLink"
+        className="page-link"
+        onClick={() => setCurrentPage(page)}
```

Comment:
Using an anchor tag (`<a>`) with an `href` attribute that points to a fragment identifier can cause the page to jump to that anchor when clicked. If the intent is not to navigate but to perform a page change without a jump, consider preventing the default action in the `onClick` event handler.